Download button error.(Android 5.1)

### Steps to reproduce

1.Click the download video button.
2.An error occurred.

### Expected behavior

There should be no error.

### Actual behavior

java.lang.RuntimeException: Unable to start service com.github.libretube.services.DownloadService@24d4726 with Intent { cmp=com.github.libretube/.services.DownloadService (has extras) }: java.lang.SecurityException: need WRITE_EXTERNAL_STORAGE permission to use DESTINATION_FILE_URI: uid 10040 does not have android.permission.WRITE_EXTERNAL_STORAGE.
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2951)
	at android.app.ActivityThread.access$2100(ActivityThread.java:151)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1411)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5351)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:947)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:742)
Caused by: java.lang.SecurityException: need WRITE_EXTERNAL_STORAGE permission to use DESTINATION_FILE_URI: uid 10040 does not have android.permission.WRITE_EXTERNAL_STORAGE.
	at android.os.Parcel.readException(Parcel.java:1546)
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:239)
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:180)
	at android.content.ContentProviderProxy.insert(ContentProviderNative.java:502)
	at android.content.ContentResolver.insert(ContentResolver.java:1207)
	at android.app.DownloadManager.enqueue(DownloadManager.java:957)
	at com.github.libretube.services.DownloadService.downloadManagerRequest(DownloadService.kt:8)
	at com.github.libretube.services.DownloadService.onStartCommand(DownloadService.kt:28)
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:2934)
	... 9 more

### LibreTube version

0.6.0

### Android version

Android 5.1

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/util/DownloadHelper.kt
+++ b/app/src/main/java/com/github/libretube/util/DownloadHelper.kt
@@ -1,13 +1,21 @@
 package com.github.libretube.util
 
 import android.content.Context
+import android.os.Build
+import com.github.libretube.BuildConfig
 import com.github.libretube.constants.DownloadType
 import com.github.libretube.obj.DownloadedFile
 import java.io.File
 
 object DownloadHelper {
     private fun getOfflineStorageDir(context: Context): File {
-        return context.getExternalFilesDir(null)!!
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) return context.filesDir
+
+        return try {
+            context.getExternalFilesDir(null)!!
+        } catch (e: Exception) {
+            context.filesDir
+        }
     }
 
     fun getVideoDir(context: Context): File {

